claim-check/claim-check-producer.bal

```diff
@@ -0,0 +1,30 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/uuid;
+import ballerinax/kafka;
+
+const MRI_SCANS = "mri-scans";
+
+final http:Client awsS3Client = check new("http://bucket.s3.amazonaws.com.balmock,io");
+final kafka:Producer kafkaProducer = check new(kafka:DEFAULT_URL);
+
+service /api/v1  on new http:Listener(8080) {
+    
+    isolated resource function post scans/mri(http:Request request) returns error? {
+        string fileExt = check request.getHeader("File-Ext");
+        string patientId = check request.getHeader("Patient-Id");
```

Comment:
see if can get the file name in more standard way

Reply:
updated.